README.md

```diff
@@ -167,6 +167,28 @@ that targets Apache Spark.
   user has a limited role-based access to Kubernetes, such as on OpenShift and also that
   `radanalyticsio/spark-operator` can deploy standalone Spark clusters.
 
+### Operator Marketplace
+
+If you would like to install the operator into OpenShift 4.x using the [Operator Marketplace](https://github.com/operator-framework/operator-marketplace), simply run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: radanalyticsio-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: radanalyticsio
+  displayName: "Operators from radanalytics.io"
+  publisher: "Jirka Kremser"
+EOF
+```
```

Comment:
does this require having the olm enabled on your openshift cluster?

Reply:
yes, it requires the OLM, afaik. Could you please try if it works for you?

Reply:
btw. it will register itself with this: https://quay.io/application/radanalyticsio/radanalytics-spark?tab=releases

Reply:
i don't have an olm enabled cluster currently, but i will try to get something going.

also, i think it would be nice to just have a quick note in the doc saying that this method requires olm.